Gems/PhysX/Code/Source/PhysXCharacters/Components/CharacterControllerComponent.h

```diff
@@ -125,6 +125,13 @@ namespace PhysX
         float GetHalfForwardExtent() override;
         void SetHalfForwardExtent(float halfForwardExtent) override;
 
+        // carbonated begin aoreshko game_specific_1
+#if defined(CARBONATED)
+        void SetMaterialByName(uint32_t index, const AZStd::string& name) override;
+        void SetTag(const AZ::Crc32& tag) override;
+#endif
+        // carbonated end game_specific_1
```

Comment:
// carbonated end -> // carbonated end aoreshko
here and in other places

Reply:
We drop names from engine code, just short branch name.